src/ruby/integration/deploy_an_app_with_execjs_test.go

```diff
@@ -1,29 +0,0 @@
-package integration_test
-
-import (
-	"github.com/cloudfoundry/libbuildpack/cutlass"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("requiring execjs", func() {
-	var app *cutlass.App
-	AfterEach(func() { app = DestroyApp(app) })
-
-	BeforeEach(func() {
-		app = cutlass.New(Fixtures("with_execjs"))
-		app.SetEnv("BP_DEBUG", "1")
-	})
-
-	It("installs node and execjs", func() {
-		PushAppAndConfirm(app)
-		Expect(app.Stdout.String()).To(ContainSubstring("Installing node"))
-		Expect(app).To(HaveUnchangedAppdir("BuildDir Checksum Before Supply", "BuildDir Checksum After Supply"))
-
-		Expect(app.GetBody("/")).To(ContainSubstring("Successfully required execjs"))
-		Expect(app.Stdout.String()).ToNot(ContainSubstring("ExecJS::RuntimeUnavailable"))
```

Comment:
is this not a case we care to test for anymore?

Reply:
As far as I can tell, there's nothing special about execjs that isn't covered by other apps with javascript (`rails6_sprockets`, `rails7`, etc). It's just a library that happens to have a dependency on javascript.

This test was [added 9 years ago](https://github.com/cloudfoundry/ruby-buildpack/commit/96c7e1f63a306614779198fd054f857527959d94). There's no real context in the tracker story other than "using the latest version of nodejs".

So I wonder if this test was just the one example they used to drive out nodejs support. But we have that covered by other tests.